src/components/DataItemList/DataItemsList.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import DataItem, { DataItemsProps } from "../DataItem/DataItem";
+
+export type DataItemsListProps = {
+  itemsList: DataItemsProps[];
+  className?: string;
+}
+
+export const DataItemsList = React.forwardRef<HTMLDivElement, DataItemsListProps>(function DataItemList(
+  {itemsList, className, ...props},
+  ref,
+) {
+
+  const getItemClass = (itemIndex: number) => {
+    let itemClass;
+
+    if (itemIndex === 0) {
+      itemClass = styles.first;
+    } else if (itemIndex + 1 === itemsList.length) {
+      itemClass = styles.last;
+    } else {
+      itemClass = styles.middle;
+    }
+
+    return itemClass;
+  };
+
+  return(
+    itemsList.map((item, index) => (
+      <DataItem
+        icon={item.icon}
+        title={item.title}
+        description={item.description}
+        className={getItemClass(index)}
+      />
+    ))
+  )
+})
+
+const styles = {
+  first: css`
+    border-radius: 4px 4px 0 0;
+    border-bottom: 0;
+  `,
+  last: css`
+    border-radius: 0 0 4px 4px;
+  `,
+  middle: css`
+    border-bottom: 0;
+    border-radius: 0;
+  `
+}
+
+export default DataItemsList;
```

Comment:
Just noticed this, we already have a named export. This is not needed

Reply:
I'd prefer to have default export anyway